ui/pages/add-collectible/add-collectible.js

```diff
@@ -65,6 +69,26 @@ export default function AddCollectible() {
       );
     }
     dispatch(setNewCollectibleAddedMessage('success'));
+
+    const tokenDetails = await getTokenStandardAndDetails(
+      address,
+      null,
+      tokenId.toString(),
+    );
+
+    trackEvent({
+      event: 'Token Added',
+      category: 'Wallet',
+      sensitiveProperties: {
+        token_contract_address: address,
+        token_symbol: tokenDetails?.symbol,
+        tokenId: tokenId.toString(),
+        asset_type: ASSET_TYPES.COLLECTIBLE,
+        token_standard: tokenDetails?.standard,
+        source: 'custom',
```

Comment:
just a question what does `source: 'custom'` mean?

Reply:
custom means its added manually. possible options right now are 'custom' | 'dapp' | 'list' | 'detected'

Reply:
@segun 

should we set this to `detected`? In [#13477](https://github.com/MetaMask/metamask-extension/issues/13477), one of our tasks is to 
>  Set source property to detected on the new token added event